### PR TITLE
[one-cmds] Improve help message of one-quantize

### PR DESCRIPTION
**Pull request by jinevening**
This improves help message of one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/7896

### CODE REVIEW

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
         '--granularity',
         type=str,
-        help='quantize granularity (supported: layer, channel, default=layer)')
+        help='quantization granularity (supported: layer, channel, default=layer)')
     quantization_group.add_argument(
         '--input_type',
         type=str,
-        help='input data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op wil be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
```
Comment (mhs4670go) [inline compiler/one-cmds/one-quantize:96]:
```suggestion
        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op will be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
```

File: compiler/one-cmds/one-quantize [hunk 2]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
+        help=
+        'maximum percentile (0.0~100.0, default=99.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
+    )
     quantization_group.add_argument(
         '--mode',
         type=str,
-        help='record mode (supported: percentile/moving_average, default=percentile)')
+        help=
+        'calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). percentile mode uses the n-th percentiles as min/max values. moving_average mode records the moving average of min/max.'
+    )
```
Comment (mhs4670go) [inline compiler/one-cmds/one-quantize:121]:
```suggestion
        'calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). Percentile mode uses the n-th percentiles as min/max values. moving_average mode records the moving average of min/max.'
    )
```

File: compiler/one-cmds/one-quantize [hunk 3]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
         '--granularity',
         type=str,
-        help='quantize granularity (supported: layer, channel, default=layer)')
+        help='quantization granularity (supported: layer, channel, default=layer)')
     quantization_group.add_argument(
         '--input_type',
         type=str,
-        help='input data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op will be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:96]: (optional) `Quantize` -> `QUANTIZE`

File: compiler/one-cmds/one-quantize (hunk 2 above)
Comment (seanshpark) [reply compiler/one-cmds/one-quantize:121]: came from `percentile/moving_average`...
(optional) maybe use 'percentile'/'moving_average' with `'` for argument names...

Comment (mhs4670go) [reply compiler/one-cmds/one-quantize:121]: Oh, I see.

File: compiler/one-cmds/one-quantize (hunk 3 above)
Comment (jinevening) [inline compiler/one-cmds/one-quantize:96]:
```suggestion
        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). QUANTIZE Op will be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
```

File: compiler/one-cmds/one-quantize [hunk 4]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
+        type=str,
+        help=
+        'maximum percentile (0.0~100.0, default=99.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
+    )
     quantization_group.add_argument(
         '--mode',
         type=str,
-        help='record mode (supported: percentile/moving_average, default=percentile)')
+        help=
+        'calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). percentile mode uses the n-th percentiles as min/max values. moving_average mode records the moving average of min/max.'
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:120]:
```suggestion
        "calibration algorithm for post-training quantization (supported: percentile/moving_average, default=percentile). 'percentile' mode uses the n-th percentiles as min/max values. 'moving_average' mode records the moving average of min/max."
```

File: compiler/one-cmds/one-quantize (hunk 2 above)
Comment (jinevening) [reply compiler/one-cmds/one-quantize]: @seanshpark is right :) I added `'` for argument names.

File: compiler/one-cmds/one-quantize [hunk 5]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
-        help='input data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of inputs of quantized model (supported: uint8, int16, default=quantized_dtype). QUANTIZE Op will be inserted at the beginning of the quantized model if input_type is different from quantized_dtype.'
+    )
     quantization_group.add_argument(
         '--output_type',
         type=str,
-        help='output data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of outputs of quantized model (supported: uint8, int16, default=quantized_dtype). Quantize Op will be inserted at the end of the quantized model if output_type is different from quantized_dtype.'
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:102]: maybe revise `Quantize` to `QUANTIZE` too

Comment (jinevening) [inline compiler/one-cmds/one-quantize:102]:
```suggestion
        'data type of outputs of quantized model (supported: uint8, int16, default=quantized_dtype). QUANTIZE Op will be inserted at the end of the quantized model if output_type is different from quantized_dtype.'
```

File: compiler/one-cmds/one-quantize [hunk 6]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
+        type=str,
+        help=
+        'minimum percentile (0.0~100.0, default=1.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
+    )
     quantization_group.add_argument(
-        '--max_percentile', type=str, help='maximum percentile (0.0~100.0, default=99.0)')
+        '--max_percentile',
+        type=str,
+        help=
+        'maximum percentile (0.0~100.0, default=99.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:114]:
```suggestion
        'maximum percentile (0.0~100.0, default=99.0). Algorithm parameter for calibration. This is valid when calibration algorithm is percentile.'
```

File: compiler/one-cmds/one-quantize [hunk 7]
```diff
@@ -80,27 +83,42 @@ def _get_parser():
...
-        help='output data type (supported: uint8, int16, default=quantized_dtype)')
+        help=
+        'data type of outputs of quantized model (supported: uint8, int16, default=quantized_dtype). QUANTIZE Op will be inserted at the end of the quantized model if output_type is different from quantized_dtype.'
+    )
     quantization_group.add_argument(
-        '--min_percentile', type=str, help='minimum percentile (0.0~100.0, default=1.0)')
+        '--min_percentile',
+        type=str,
+        help=
+        'minimum percentile (0.0~100.0, default=1.0). Algorithm parameter for calibration. This is valid when record mode is percentile.'
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:108]:
```suggestion
        'minimum percentile (0.0~100.0, default=1.0). Algorithm parameter for calibration. This is valid when calibration algorithm is percentile.'
```

**Review by seanshpark — APPROVED** [review]
LGTM thank you

**Review by mhs4670go — COMMENTED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]

**Pull request merged**